src/libsync/discovery.cpp

```diff
@@ -1292,7 +1292,19 @@ void ProcessDirectoryJob::processFileAnalyzeLocalInfo(
         if (base.isE2eEncrypted()) {
             // renaming the encrypted folder is done via remove + re-upload hence we need to mark the newly created folder as encrypted
             // base is a record in the SyncJournal database that contains the data about the being-renamed folder with it's old name and encryption information
-            item->_isEncrypted = static_cast<SyncFileItem::EncryptionStatus>(base._isE2eEncrypted);
+            switch (base._e2eEncryptionStatus)
+            {
+            case SyncJournalFileRecord::EncryptionStatus::Encrypted:
+            case SyncJournalFileRecord::EncryptionStatus::EncryptedMigratedV1_2Invalid:
+                item->_e2eEncryptionStatus = SyncFileItem::EncryptionStatus::Encrypted;
```

Comment:
@mgallien should not you use a method `SyncFileItem::EncryptionStatus fromDbEncryptionStatus(SyncJournalFileRecord::EncryptionStatus encryptionStatus)` here?